libs/filabridge/include/private/filament/SamplerBindingMap.h

```diff
@@ -53,36 +35,22 @@ class SamplerBindingMap {
 
     // Given a valid Filament binding point and an offset within the block, returns true and sets
     // the output argument 'globalOffset' to the globally unique binding index.
```

Comment:
This comment is now out of date, it should explain that the return value is the global offset.